test/backup.py

```diff
@@ -361,6 +365,21 @@ def _test_backup(self, tablet_type):
     # now bring up the other slave, letting it restore from backup.
     self._restore(tablet_replica2, tablet_type=tablet_type)
 
+    tablet_replica2.kill_vttablet()
+
+    # now delete the data dir
+    shutil.rmtree(os.path.join(tablet_replica2.tablet_dir, 'data'))
+
+    # create .restore file
```

Comment:
Since `_restore` deletes the whole tablet tree, I don't think this is actually testing the case when a `.restore` file exists.

Reply:
Aha! I thought the test worked too easily. I'll rework it.